### PR TITLE
feat: Add RemoteSwitch showing deletion profile status 

### PR DESCRIPTION
**Pull request by Hantex9**
<h3 align="center">⚠️ This PR depends on #1 ⚠️</h3>
<p align="center">Please review first #1 in order to avoid duplicate review work </p>

## Short description
This PR introduces a Switch into `NewProfileScreen` that indicates the deletion profile status, if the user previously requested a profile deletion and the request is PENDING it is toggled, otherwise it is untoggled.

## List of changes proposed in this pull request
- Added `RemoteSwitch` component into `ProfileContentView` sub-component into main screen;
- Added a `isUserDataProcessingDeleteSelector` memoized selector which returns a pot that is true if the deletion process status is _PENDING_ otherwise it is false
- Added localized labels into _it_ and _en_ language

## How to test
- Run the app on the emulator/simulator;
- Login into the app;
- Press on the _profile_ bottom tab;
- Actually there is not a "profile deletion flow request", so you can check only if in the first render of the Screen you are able to see a spinner indicator where is placed the RemoteSwitch

## Preview
<p align="center">

</p>

### CODE REVIEW

File: ts/screens/profile/NewProfileScreen.tsx [hunk 1]
```diff
@@ -99,9 +110,13 @@ const NewProfileScreen = () => {
           subTitle={getValueFromNewProfilePot("email")}
           leftIcon={EmailIcon}
           hideIcon
-          isLastItem
           testID="email"
         />
+        <PreferencesListItem
+          title={I18n.t("profile.data.deletion.title")}
+          description={I18n.t("profile.data.deletion.description")}
+          rightElement={<RemoteSwitch value={isUserDataProcessingDeletePot} />}
```
Comment (CrisTofani) [inline ts/screens/profile/NewProfileScreen.tsx:118]:
```suggestion
          rightElement={<RemoteSwitch value={isUserDataProcessingDeletePot} onValueChange={_ => constVoid} />}
```
just as a suggestion to if you want you can add a placeholder function which won't actually cause any side effects but will work only as a TODO to remove in a future implementation

**Review by CrisTofani — COMMENTED** [review]
see it as a suggestion, this actually won't add anything more to the code. I see you went through implementing the next functionality, so this will be only a comment for future documentation 😄

**Review by CrisTofani — APPROVED** [review]
LGTM

File: ts/screens/profile/NewProfileScreen.tsx (hunk 1 above)
Comment (Hantex9) [reply ts/screens/profile/NewProfileScreen.tsx:118]: Thanks!
Next time I will use it whenever I need to add a placeholder function 🙌

**Pull request merged**